exercises/knapsack/canonical-data.json

```diff
@@ -12,6 +12,17 @@
       },
       "expected": 0
     },
+    {
+      "uuid": "3993a824-c20e-493d-b3c9-ee8a7753ee59",
+      "reimplements": "a4d7d2f0-ad8a-460c-86f3-88ba709d41a7",
```

Comment:
Do we need a re-implements when the expected generated test code shouldn't change?

Reply:
Well, the CI tests fail if I change the existing test case :shrug:

Reply:
It's technically a breaking change, so yeah the reimplements makes sense here I think.